docs/source/models/concept_models/vivarium_mncnh_portfolio/ai_ultrasound_module/module_document.rst

```diff
@@ -41,11 +41,246 @@ AI Ultrasound Module
+  - Definition of "unknown", "unreliable", or "uncertain"
+  - Proportion of pregnancies classified as unknown/unreliable/uncertain
+
+.. warning::
+
+  Many studies that compare LMP gestational age dating accuracy to other methods have 
+  inclusion criteria of "reliable" or "certain" LMP estimates, meaning that the LMP accuracy 
+  in these studies will be overestimated relative to its actual use. We should be careful in 
+  evaluating the inclusion/exclusion criteria of studies that assess LMP gestational age 
+  dating accuracy.
+
+  Additionally, prospectively collected menstrual period dates (via calendar tracking or 
+  tracking via an app) will be more accurate than retrospectively collected LMP dates due to 
+  recall bias. Given the differential practices around prospective tracking (due to 
+  differential availability of apps and/or cultural differences) between high and low 
+  resource settings, we should aim to have the accuracy of LMP GA dating informed from 
+  studies performed in populations similar to the ones we are modeling.
+
+1.1.1.3 Fundal height measurements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Measurements of fundal height, the distance between the pubic bone and umbillicus can be used 
+to approximate fetal growth after 20 weeks of gestation, with an expected centimeter for 
+every week of gestational age (27 cm for 27 weeks, for example). This method is limited in 
+that it cannot distinguish between variation in fetal size due to variation in gestational 
+age or factors that cause small for gestational age (like intrauterine growth restriction). 
+It is additionally limited in that it may vary due to differences in maternal stature and 
+becomes less reliable in late pregnancy when the fetus "drops" into the pelvis 
+("lightening"), usually 2-4 weeks before labor. It is also subject to measurement error due 
+to provider skill and resources (it is reported to be measured with fingers if measuring 
+tapes are not available).
+
+1.1.1.4 Newborn assessment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In cases in which a reliable estimate of gestational age is not available at the onset of 
+labor, gestational age at birth can be assessed by evaluating the newborn according to 
+physical and neuromuscular signs. The New Ballard and Dubowitz scoring systems are a 
+measurement tools meant to be performed in the first hours after birth that can estimate a 
+neonate's maturity and distinguish between small for gestational age term infants from truly 
+preterm infants [Lee-et-al-2017]_.
+
+1.1.2 Clinical practice overview
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The WHO recommends that all pregnancies undergo at least one ultrasound for identification of 
+certain complications as well as gestational age dating. However, ultrasound technology is 
+not available at all ANC visits across LMICs. Additionally, ANC attendance during the first 
```

Comment:
I should know this, but has someone already made estimates of US coverage by year and geography?

Reply:
I was involved in the selection of these parameters so I don't have the full context, but it looks like our model is relying on non-representative reports for baseline coverage of ultrasound: https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_mncnh_portfolio/ai_ultrasound_module/module_document.html#baseline-coverage

Although the note also implies that there is data on this available through DHS so maybe that would be an improvement? I just looked back on notes from Annie on what her team estimates and in her list from April 2024 it did not include ultrasound coverage at ANC.